client/src/components/Banner/Banner.css

```diff
@@ -0,0 +1,38 @@
+.banner{
+  padding: 0;
+  margin: 0 auto;
+  width: 100%;
+  width: 100vw;
+  height: 231px;
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  text-align: center;
+  font-family: var(--default-font);  
+  font-size: 48px;
+  font-style: normal;
+  font-weight: normal;
+  color: #FAFAFA;
+  line-height: 72px;
+}
+
+.Banner__header{
+  padding-top: 10px;
+}
+
+.Banner__header2{
+  text-align: left;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.Banner__header3{
+  margin: 0 auto;
+}
```

Comment:
Can you find semantic names for these? We never want to put numbers in our css classes